eng/common/TestResources/New-TestResources.ps1

```diff
@@ -215,11 +215,18 @@ try {
     $PSBoundParameters['BaseName'] = $BaseName
 
     # Try detecting repos that support OutFile and defaulting to it
-    if (!$CI -and !$PSBoundParameters.ContainsKey('OutFile') -and $IsWindows) {
+    if (!$CI -and !$PSBoundParameters.ContainsKey('OutFile')) {
         # TODO: find a better way to detect the language
```

Comment:
@scbedd noted on the previous PR: https://github.com/Azure/azure-sdk-for-rust/pull/2487#discussion_r2045579845